### ISSUE TITLE
custom kernel

### ISSUE THREAD

**Issue by pribadihcr** [apache/incubator-mxnet#10832]
HI,
How to make custom kernel for mxnet convolution like the following tf version:
def gauss_kernel(kernlen=21, nsig=3, channels=1):
        interval = (2*nsig+1.)/(kernlen)
        x = np.linspace(-nsig-interval/2., nsig+interval/2., kernlen+1)
        kern1d = np.diff(st.norm.cdf(x))
        kernel_raw = np.sqrt(np.outer(kern1d, kern1d))
        kernel = kernel_raw/kernel_raw.sum()
        out_filter = np.array(kernel, dtype = np.float32)
        out_filter = out_filter.reshape((kernlen, kernlen, 1, 1))
        out_filter = np.repeat(out_filter, channels, axis = 2)
        return out_filter

def blur(x):
        kernel_var = gauss_kernel(21, 3, 3)
        return tf.nn.depthwise_conv2d(x, kernel_var, [1, 1, 1, 1], padding='SAME')

**Comment by roywei** [comment]
@squidszyd Could you also raise your question at https://discuss.mxnet.io/ and stackoverflow?
@sandeep-krishnamurthy could you help to add label Question? Thanks

**Comment by sandeep-krishnamurthy** [comment]
Please ask usage related questions at https://discuss.mxnet.io/ for more people being able to answer, search and refer in future. We want to keep GitHub for issues/bugs.
